### DISCUSSION TITLE
can sync mail from gmail to maddy via imapsync

### DISCUSSION THREAD

**Discussion by vtolstov** [emersion/maddy#471] [Q&A] [answered]
```
Err 20/50: - msg &BC8EPQQ0BDUEOgRB-/24310 {35630523} could not append ( Subject:[=?UTF-8?B?0JXQu9C60LAgNQ==?=], Date:["12-Jan-2018 17:37:32 +0000"], Size:[35630523], Flags:[\Seen] ) to folder &BC8EPQQ0BDUEOgRB-: 6960 NO [TOOBIG] Message size exceeding limit
```

How to increase message size limit in maddy?

**Comment by foxcpp** [comment, marked as answer]
Size of messages written via IMAP is limited via `appendlimit` directive for the [storage.imapsql module](https://github.com/foxcpp/maddy/blob/dev/docs/reference/storage/imapsql.md). Default is 32 MiB. Message that caused the error is almost 33.9 MiB.

To double the limit:
```
storage.imapsql local_mailboxes {
  ...
  appendlimit 64M
}
```

**Reply by vtolstov** [reply]
thanks this helps a lot

**Comment by vtolstov** [comment]
do you have some roadmap/guidelines - i want to help with development =)
Mostly because now i'm use it for own emails after go away from gmail, and because i want to use it as package in one of my projects
